src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -161,17 +162,31 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
   uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
-
-  return absl::StrFormat(
+  std::string tooltip = absl::StrFormat(
```

Comment:
Since you are splitting it, consider giving each part a different name.

Reply:
It was like that at first, split into `description`, `woker_info`, and `time`. Henning told me it might be better to make it a single string and use `+=` so I went with that.

Reply:
ok. I think I prefer it being splitted but since you've already considered this, I'm ok.